src/main/views/macro/financialDetails.njk

```diff
@@ -0,0 +1,188 @@
+{% from "table.njk" import tableStart, tableEnd %}
+
+{% macro heading(label) %}
+  <h3 class="heading-small form-group-related">{{ t(label) }}</h3>
+{% endmacro %}
+
+{% macro row(label, value, bottomBorder = false) %}
+  <div class="{% if bottomBorder %}bottom-border{% else %}no-bottom-border{% endif %}">
+    <div class="column-one-half {% if not label %}if-empty{% endif %}">
+      <span>{{ t( label ) }}</span>
+    </div>
+    <div class="column-one-half {% if not value %}if-empty{% endif %}">
+      <span>{{ value }}</span>
+    </div>
+  </div>
+{% endmacro %}
+
+{% macro bankAccountsSection(bankAccounts) %}
+  {% for bankAccount in bankAccounts %}
+    {% if bankAccounts.length === 1 %}
+      {{ row(label = 'Type of account', value = bankAccount.type | renderBankAccountType) }}
```

Comment:
are these filters useful?
i.e. don't most of these models have `displayValue` on them?

Reply:
The approach with `displayValue` is just temporary. It will remain there until we remove these methods from TS classes. The reasoning behind using filters instead of constants with `displayValue` is that we want to separate UI from data.